src/games/strategy/common/swing/SwingComponents.java

```diff
@@ -30,12 +32,15 @@
 import javax.swing.border.Border;
 import javax.swing.border.EmptyBorder;
 
+import com.google.common.collect.Sets;
 import games.strategy.engine.framework.map.download.DownloadFileDescription;
 import games.strategy.net.DesktopUtilityBrowserLauncher;
 import games.strategy.triplea.UrlConstants;
 
 public class SwingComponents {
 
+  private static Set<String> visiblePrompts = Sets.newHashSet();
```

Comment:
Because its native Java and just as straightforward/concise. No reason to use a library for that.

From the Guava docs, "Note for Java 7 and later: this method is now unnecessary and should be treated as deprecated. Instead, use the HashSet constructor directly, taking advantage of the new "diamond" syntax." https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/collect/Sets.html#newHashSet()